src/res/MacOSXBundleInfo.plist.in

```diff
@@ -5,7 +5,7 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>English</string>
 	<key>CFBundleExecutable</key>
-	<string>${MACOSX_BUNDLE_EXECUTABLE_NAME}</string>
+	<string>cslol-manager-elevate</string>
```

Comment:
```suggestion
	<string>${MACOSX_BUNDLE_EXECUTABLE_NAME}-elevate</string>
```
Maybe still keep variable name but append -elevate to it.
Idk

Reply:
:+1: